content/getting-started/connect-the-pim-old/step-2.md

```diff
@@ -0,0 +1,112 @@
+
+::: info
+_Best practice_  
+Always create a dedicated user for each one of your connectors.  
+We also strongly recommend you not to use a real user, as a UI user will never do the same actions than an API user.
+:::
+
+## Create an API user role
+
+Then, we are going to create a dedicated user role for your connector. We will name it `My ERP connector role`, for our tutorial example. 
+
+1. Click on `System` menu.
+1. Click on the `Role` submenu. 
+1. From here, you should be able to create a role, click on `Create role`.
+![Role creation](/img/getting-started/role-creation.png)
+1. Once in the role form, give a name to the role you are creating: `My ERP connector role`.
+![Role naming](/img/getting-started/role-naming-my-erp-connector.png)
+1. Go to the `Web API Permissions` tab and activate the access to the Web API by clicking on `Overall Web API access`.
+![Permission activation](/img/getting-started/permission-activation.png)
+1. Still in this tab, select the other permissions needed for your connector's API calls to work.  
+_In the case of our ERP example, we don't need to activate other permissions, as the connector will only create products and this permission is already granted by the `Overall Web API access` above._
+1. Click on the `Users` tab and tick the box next to the `myERPuser` to associate your API user to this new role.
+![Permission assignation](/img/getting-started/permission-assignation-my-erp-user.png)
+1. Click on the `Save` button.
+
+::: info
+_Best practice_  
+Do not - under any condition :wink: - re-use a user role that is assigned to real users. Indeed, we highly recommend you to create a dedicated user role for each one of your connectors.  
+This way, you can give this user role, exactly the API permissions needed for your connector to work.
+:::
+
+## Get your client ID and secret
+
+Next, you will need to generate a client ID and a secret for your connection.
+
+::: info
+_Best practice_  
+Generate one set of client ID/secret by connector. Indeed, we advise you not to share them between your connectors.  
+It will simplify the management of these credentials. For example, in case one of the secret leaks, it won't impact all your connectors at once. :wink:
+:::
+
+Depending on your PIM version (v1.7 or v2/v3), there are two methods to generate your client ID and secret.
+
+If you are using a v2 or V3 PIM, [create an API connection](#by-creating-an-API-connection).  
+If you are using a v1.7, [launch a command line](#by-command-line-for-v17-pim-only).
+
+### By creating an API connection
+
+We will create an API connection. We will call it, for the purpose of this tutorial: `ERP`
+
+::: warning
+This feature is only available since the 2.0 version. If you are trying to use the API on a 1.7 PIM, please have a look to the [second method below](#get-your-credentials-via-command-line-).
```

Comment:
```suggestion
This feature is only available since the 2.0 version. If you are trying to use the API on a 1.7 PIM, please take a look at the [second method below](#get-your-credentials-via-command-line-).
```